Dockerfile

```diff
@@ -34,9 +34,9 @@ COPY --from=builder /root/.local /usr/local
 
 RUN apt-get update -y \
     && apt-get install -y --no-install-recommends libeccodes-tools \
-    && python3 -m pip install --no-cache-dir nose \
+    && python3 -m pip install --no-cache-dir nose smrf[tests] \
     && nosetests -vv --exe smrf \
-    && python3 -m pip uninstall -y nose \
+    && python3 -m pip uninstall -y nose smrf[tests] \
```

Comment:
I think that should make the Docker built happy again. :crossed_fingers:

Reply:
This was trying to install smrf from pypi so it didn't. Updated the install to add the extras and the docker builds.